.gitignore

```diff
@@ -12,6 +12,7 @@
 /tmp/*
 !/log/.keep
 !/tmp/.keep
+/db/*.sqlite3
```

Comment:
I generated the rails app with postgres and forgot to add this to the gitignore when I installed sqlite. This pr removes the two sqlite files. 

master:
https://github.com/cerner/terra-ui.com/tree/master/db

this branch:
https://github.com/cerner/terra-ui.com/tree/consume-terra-clinical-site/db